AtReconstruction/AtPatternRecognition/AtTrackFinderTC.cxx

```diff
@@ -87,12 +94,24 @@ std::unique_ptr<AtPatternEvent> AtPATTERN::AtTrackFinderTC::FindTracks(AtEvent &
    add_clusters(cloud_xyz, cl_group, opt_params.is_gnuplot());
 
    // Post processing
-   // process_pointcloud(cloud_xyz, 25, 0);
+   //process_pointcloud(cloud_xyz, 25, 0);
 
    // Adapt clusters to AtTrack
    return clustersToTrack(cloud_xyz, cl_group, event);
 }
 
+const double tolerance = 3.0;
```

Comment:
This should be a parameter set on a per-instance level (no recompiling if changed - will work for all experiments)

Reply:
It looks like this also lives int he global namespace?